compiler-core/src/language_server/engine.rs

```diff
@@ -354,6 +355,8 @@ where
             completions.push(type_completion(None, name, type_));
         }
 
+        let root_package = self.compiler.project_compiler.config.name.clone();
```

Comment:
I don't think we need to clone here, using a `&str` will be fine. A helper function `self.root_package_name()` could be good too